test/e2e/README.md

```diff
@@ -1,6 +1,6 @@
 # E2E Tests
 
-Prow will run `./e2e-tests.sh`.
+Prow will run `./e2e-tests.sh` with authentication mechanism using Kubernetes Secrets and `./e2e-wi-tests.sh` with authentication mechanism using Workload Identity.
```

Comment:
It's better to point out that Workload Identity is GKE specific and add the link https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity here